### ISSUE TITLE
for future development: optional properties controlled via management and/or with author override

### ISSUE THREAD

**Issue by ronm123** [thexerteproject/xerteonlinetoolkits#1139]
Adding this to start a list of things we could add as optional defaults controlled via management and possibly with author override control too.

- [ ] Controlling SCORM tracking alert shown to users

In management I think it would have to be a two stage option like we did for the social icons e.g. option to set off by default and option to allow author override or not.

- [ ] open ckeditor toolbar by default

Should this be controlled via management or should it just be on by default anyway?

- [ ] User options

Tom's comment: Or perhaps see whether we can implement user options. Should be simple enough to store them in a json framgment in the logindetails table.

**Comment by FayCross** [comment]
I would prefer ckeditor toolbar not to be open by default - on some pages there's loads of text fields and it would start to be really cluttered. I don't really see the value in setting this in management either but I guess I just wouldn't use it if it was added!

I've added another to the list - accessibility link

**Comment by FayCross** [comment]
Some comments from a card in Trello re. accessibility statement copied here so they don't get lost:

We currently have the default link in the following files:

- application.js
- site player rloObject.html
- extend.php
- management.php
- colourChanger.html
- login_library.php

Two areas of enhancement:
1. change the hardcoded link in all the files to pull and use the link value from management
2. ensure that if the optional property has been changed this still overrides the value set via management

The 4 properties that can be changed via optional properties (which would therefore need to also be editable via management) are:

- wcagLink
- wcagLinkTitle
- wcagAlt
- wcagHide
